docs/en/integrations/data-visualization.md

```diff
@@ -39,6 +39,7 @@ Now that your data is in ClickHouse, it's time to analyze it, which often involv
 | [Deepnote](./data-visualization/deepnote.md)                                            | Native connector    | ✅     | ✅     |            |
 | [Explo](./data-visualization/explo-and-clickhouse.md)                                   | Native connector    | ✅     | ✅     |            |
 | [Grafana](./data-visualization/grafana-and-clickhouse.md)                               | ClickHouse official connector  | ✅     | ✅     |            |
+| [Hashboard](./data-visualization/hashboard-and-clickhouse.md)                             | Native connector  | ✅     | ✅     |      |
```

Comment:
could you add Hashboard to https://github.com/ClickHouse/clickhouse-docs/blob/60272cea481e0a73b6443348a3bc660e033ff8f6/docs/en/integrations/index.mdx ?

Reply:
Done 👍